docs/specs.md

```diff
@@ -26,6 +26,8 @@ We recommend that you always use the latest **stable** specification for your op
   - [v1.1.0](config-flatcar-v1_1.md)
   - [v1.0.0](config-flatcar-v1_0.md)
 - OpenShift (`openshift`)
+  - [v4.19.0](config-openshift-v4_19.md)
+  - [v4.18.0](config-openshift-v4_18.md)
```

Comment:
As we'll have to update this, can we split the 4.18 fix in another commit?

Reply:
done!